cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/generator/GeneratorWrapperFactory.java

```diff
@@ -240,4 +280,50 @@ private static CobiGen initializeGenerator() throws InvalidConfigurationExceptio
           e);
     }
   }
+
+  /**
+   * Initializes the {@link CobiGen} with the correct configuration, even if old templates exists
```

Comment:
Better: Initializes the {@link CobiGen} with the correct configuration, even if monolithic templates exists